File: frontend/frontend/src/app/interface/RatingResponse.ts
export interface RatingResponse {
  id: number;
  raterUserId: number;
  ratedUserId: number;
  rating: number;
  comment: string;
  creationDate: string;
}

// {
//   "raterUserId": 1,
//   "ratedUserId": 2,
//   "rating": 5,
//   "comment": "Sehr gute Erfahrung!",
//   "creationDate": "2026-04-15T12:00:00"
// }

File: backend/src/main/java/de/neighbourly/backend/dto/RatingResponse.java
package de.neighbourly.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDateTime;
import java.util.Date;

@Getter
@AllArgsConstructor
public class RatingResponse {
    private long id;
    private long raterUserId;
    private long ratedUserId;
    private int rating;
    private String comment;
    private LocalDateTime creationDate;
}